### PR TITLE
Adding budget/description to json example

### LINKED ISSUE #336: Add description field to budget
The CoST IDS mapping includes the following mapping for funding sources:

image

However, CoST West Lombok's platform captures the funding sources for a project in a single free-text field, which may combine multiple sources. For example:

Source of Funds | Translation | Notes
-- | -- | --
APBD Kabupaten Lombok Barat dan Hibah DFAT Australia | West Lombok District Budget and Australia DFAT Grant | APBDs are sub-national or local government budgets. DFAT is the Australian Government Department of Foreign Affairs and Trade.
(DAK Reguler Bidang Jalan) yang dituangkan dalam APBD Kabupaten Lombok Barat | (Regular DAK for Roads) as outlined in the APBD of West Lombok Regency | DAK is the Specific Allocation Fund, a policy instrument that enables central government to make ‘specific’ fiscal transfers to regional and district governments for physical facilities and infrastructure

We can recommend that, going forward, CoST West Lombok should capture a list of funding sources for each project, which can be modelled using `budget.budgetBreakdown` and/or the `parties` section. We can also recommend that CoST West Lombok manually cleanse historical data to split out the funding sources for each project. However, this issue is likely to occur with other CoST programmes, which typically implement many of the elements of the CoST IDS as free-text fields in their systems.

OCDS's approach, at the contracting process level, is to have a `description` field on the `Budget` object with the following description:

> A short free text description of the budget source. May be used to provide the title of the budget line, or the programme used to fund this project.

**Proposal**

Add a `description` field to `budget` with the following description:

> A short free-text description of the budget source(s). May be used to provide the title of the budget line, or the programme used to fund this project.

**Alternative**

Another option is to add a single `BudgetBreakdown` object to `budget.budgetBreakdown` and set its `.description` to the free-text description of the funding sources. However, semantically, this is not a breakdown of the budget.

**References:**

* [Overview of Indonesia's Budgeting Process](https://seknasfitra.org/analisis/tentang-anggaran/?lang=en)
* [Australian Government Department of Foreign Affairs and Trade](https://www.dfat.gov.au/)
* [A Regulatory Analysis of The Specific Allocation Fund (DAK) and Horizontal Equalization in Indonesia](https://smeru.or.id/en/content/regulatory-analysis-specific-allocation-fund-dak-and-horizontal-equalization-indonesia)
* [DAK Fisik: Encouraging Government to Provide Quality Basic Services](https://kompak.or.id/en/article/dak-fisik-encouraging-government-provide-quality-basic-services)

### PR DESCRIPTION
**Pull request by odscrachel**
**Related issues**

Closes #336
Completes work started in #365

**Description**

Adding budget/description field to blank and example json

**Merge checklist**

- [x] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)

If there are changes to `project-schema.json` or `project-package-schema.json`:

- [x] Update the examples:
  - [x] `docs/examples/example.json`
  - [x] `docs/examples/blank.json`
- [ ] Run `./manage.py pre-commit` to update `docs/_static/i8n.csv`

If you added a new definition to the schema, update `docs/reference/schema.md`:

- [ ] Add an entry to the components section
- [ ] Update the `:collapse:` parameter of the `jsonschema` directive for any schemas or sub-schemas that reference the new definition 

If you added a new codelist:

  - [ ] Add an entry to `docs/reference/codelists.md`

### CODE REVIEW

**Review by duncandewhurst — APPROVED** [review]

File: docs/reference/changelog.md
Comment (jpmckinney) [inline docs/reference/changelog.md]: This is mainly a follow-up to #365, so let's put it on the same line.

```suggestion
* [#365](https://github.com/open-contracting/infrastructure/pull/365) [#386](https://github.com/open-contracting/infrastructure/pull/386) - add description field to budget.
* [#367](https://github.com/open-contracting/infrastructure/pull/367) - add approval date to budget breakdown.
* [#368](https://github.com/open-contracting/infrastructure/pull/368) - clarify contracting processes id description.
```

**Comment by jpmckinney** [comment]
Can be merged if suggestion is merged.

**Comment by duncandewhurst** [comment]
@jpmckinney the 0.9-dev branch [failed to build](https://github.com/open-contracting/infrastructure/actions/runs/4698469616/jobs/83307844380) after this PR was merged. It looks like something went wrong with deploy-docs.sh. Please can you advise what to do? Thanks!

**Comment by jpmckinney** [comment]
We changed servers, but I think forgot to copy the SSH credentials over. I'll address it.

**Comment by jpmckinney** [comment]
There was some missing CORS configuration, after moving Elasticsearch behind Apache. Works now!

**Comment by duncandewhurst** [comment]
Brill, thanks!

**Pull request merged**
